client/components/main.js

```diff
@@ -22,13 +22,16 @@ const Main = (props) => {
             ? <div>
               {/* The navbar will show these links after you log in */}
               <Link to="/home">Home</Link>
+              <Link to="/watches">Watches</Link>
```

Comment:
I don't think so, because we're conditionally rendering one div and all its children or the other div and all its children. We could consider making the watches/cart (any other shared stuff) into a component on refactor.